gallery/__init__.py

```diff
@@ -84,6 +84,8 @@
         None,
         app.config.get("EBOARD_UIDS", "").split(","),
         app.config.get("RTP_UIDS", "").split(","),
+        app.config.get("ORGANIZER_UIDS", "").split(","),
+        app.config.get("ALUMNI_UIDS", "").split(","),
```

Comment:
This doesn't seem to be used anywhere else in your PR, maybe remove it? Unless it's WIP

Reply:
Ahh that makes sense, thanks. Doesn't look like it -- @jabbate19 could you add them to the config samples for clarity?